src/models/dynamic_inverter.jl

```diff
@@ -26,71 +26,71 @@ mutable struct DynamicInverter{
     states::Vector{Symbol}
     ext::Dict{String, Any}
     internal::InfrastructureSystemsInternal
-    function DynamicInverter(
-        number::Int64,
-        name::String,
-        bus::Bus,
-        ω_ref::Float64,
-        V_ref::Float64,
-        P_ref::Float64,
-        Q_ref::Float64,
-        MVABase::Float64,
-        converter::C,
-        outer_control::O,
-        inner_control::IC,
-        dc_source::DC,
-        freq_estimator::P,
-        filter::F,
-        ext::Dict{String, Any} = Dict{String, Any}(),
-    ) where {
-        C <: Converter,
-        O <: OuterControl,
-        IC <: InnerControl,
-        DC <: DCSource,
-        P <: FrequencyEstimator,
-        F <: Filter,
-    }
+end
 
-        n_states = (
-            converter.n_states +
-            outer_control.n_states +
-            inner_control.n_states +
-            dc_source.n_states +
-            freq_estimator.n_states +
-            filter.n_states
-        )
+function DynamicInverter(
```

Comment:
The old inner constructor prevented me from being able to construct an object with a deserialized internal object.

Reply:
We should make consistent with the constructor for DynamicGenerator

Reply:
Fixed